diff --git a/node_modules/@types/node/index.d.ts b/node_modules/@types/node/index.d.ts
index 97cb955..e577604 100755
--- a/node_modules/@types/node/index.d.ts
+++ b/node_modules/@types/node/index.d.ts
@@ -71,7 +71,6 @@
 // NOTE: These definitions support NodeJS and TypeScript 4.9+.
 
 // Reference required types from the default lib:
-/// <reference lib="es2020" />
 /// <reference lib="esnext.asynciterable" />
 /// <reference lib="esnext.intl" />
 /// <reference lib="esnext.bigint" />
